packages/mobile/src/components/share-drawer/ShareDrawer.tsx

```diff
@@ -81,17 +91,38 @@ const useStyles = makeStyles(({ palette }) => ({
     alignItems: 'center',
     paddingVertical: 12,
     paddingHorizontal: 24
+  },
+  viewShot: {
+    position: 'absolute',
+    // Position the container off-screen (264px is the width of the whole thing)
+    right: -264 - 5
   }
 }))
 
+const stickerLoadedEventEmitter = new EventEmitter()
+const STICKER_LOADED_EVENT = 'loaded' as const
+
+const DEFAULT_DOMINANT_COLORS = ['000000', '434343']
+
 export const ShareDrawer = () => {
   const styles = useStyles()
+  const viewShotRef = useRef() as React.RefObject<ViewShot>
```

Comment:
all the sticker stuff is dope! is there a way for us to put this logic in a subcomponent, just cause share-drawer is getting pretty unweildy

Reply:
yeah i can try that out!